testrepository/commands/load.py

```diff
@@ -88,7 +93,10 @@ def run(self):
                 repo = self.repository_factory.initialise(path)
             else:
                 raise
-        testcommand = self.command_factory(self.ui, repo)
+        profiles = self.ui.options.profiles.split(',')
```

Comment:
Probably want to `.strip()` before calling `.split`, especially given the conditional below.